index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Kenya Youth conference</title>
+  <link rel="stylesheet" href="./CSS/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="top_menu">
+      <a href="#"><img src="./assets/facebook.png" alt="facebook"></a>
+      <a href="#"><img src="./assets/twitter.png" alt="twitter"></a>
+      <span class="span">English</span>
+      <span class="span">My Page</span>
+      <span class="span">Logout</span>
+    </div>
+    <nav class="nav_menu">
+      <img id="hamburger" class="hamburger" src="./assets/menu.png" alt="hamburgermenu">
+      <a class="logo" href="#"><img src="./assets/logo.png" alt="logo"></a>
+      <ul class="desktop_menu">
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Application</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="border_red"><a href="#">Presentation</a></li>
+      </ul>
+    </nav>
+    <div id="mobile_menu" class="mobile_menu">
+      <div id="close_button" class="close_button">
+        <img src="./assets/close_mobile_menu.png" alt="closeButton">
+      </div>
+      <div class="mob_div">
+        <span><a href="./about.html" >About</a></span>
+        <img src="./assets/arrow_down.png" alt="down">
+      </div>
+      <div class="mob_div">
+        <span><a href="#">Program</a></span>
+        <img src="./assets/arrow_down.png" alt="down">
+      </div>
+      <div class="mob_div">
+        <span><a href="#">Application</a></span>
+      </div>
+      <div class="mob_div">
+        <span><a href="#">Sponsor</a></span>
+      </div>
+      <div class="mob_div">
+        <span><a href="#">News</a></span>
+      </div>
+      <div class="mob_div">
+        <span><a href="#">Presentation</a></span>
+      </div>
+    </div>
+    <div class="intro_words">
+      <h1 class="title">"Hello,Kenyan Youth!"</h1>
+    <h2 class="subtitle">KENYA YOUTH SUMMIT <br>2022</h2>
+    <p class="paragraph">
+      Hey! A you aged between 18-35 years old ?,You the majority of this beautiful country called
+      Kenya, come one ,come all, as we interact and share the youth agenda,It's out time and we want to network 
+      and share our views.
+      </p>
+      <h3 class="day">2022/07/29~ sun/31</h3>
+      <h4 class="venue">@KICC ,Nairobi, Kenya </h4>
+    </div>
+  </header>
```

Comment:
as you can see in the image below the navigation is overflowing outside the content of the page I strongly suggest you fix this to improve the ux of the page

![image](https://user-images.githubusercontent.com/60612329/175751506-8c896208-1db6-4bb2-a6cf-e8692e4f011a.png)